packages/forms/src/UIForm/fields/Select.js

```diff
@@ -0,0 +1,83 @@
+import React, { PropTypes } from 'react';
+import FieldTemplate from './FieldTemplate';
+
+function getSelectedOptions(select, multiple) {
+	if (multiple) {
+		return Array.from(select.options)
+			.filter(option => option.selected)
+			.map(option => option.value);
+	}
+
+	return select.value;
+}
+
+export default function Select(props) {
+	const { id, isValid, errorMessage, onChange, schema, value } = props;
+	const { autoFocus, description, disabled, placeholder, readOnly, title } = schema;
+
+	const multiple = schema.schema.type === 'array' && schema.schema.uniqueItems;
+
+	return (
+		<FieldTemplate
+			description={description}
+			errorMessage={errorMessage}
+			id={id}
+			isValid={isValid}
+			label={title}
+			labelAfter
+		>
+			<select
+				id={id}
+				multiple={multiple}
+				autoFocus={autoFocus}
+				className="form-control"
+				disabled={disabled}
+				onChange={
+					event => onChange(event, schema, getSelectedOptions(event.target, multiple))
```

Comment:
I'm used to event, data
here it's event, schema, data
not sure about the order, should we either doing event, data, schema ?
it depends if the schema is mandatory or not for the usage.